bioblend/_tests/TestGalaxyQuotas.py

```diff
@@ -52,3 +55,22 @@ def test_delete_undelete_quota(self):
         assert response == "Deleted 1 quotas: " + self.quota_name
         response = self.gi.quotas.undelete_quota(self.quota["id"])
         assert response == "Undeleted 1 quotas: " + self.quota_name
+
+    @test_util.skip_unless_galaxy("release_19.09")  # for user purging
+    def test_update_non_default_quote(self):
+        new_username = test_util.random_string()
+        new_user_email = f"{new_username}@example.org"
+        password = test_util.random_string(20)
+        new_user = self.gi.users.create_local_user(new_username, new_user_email, password)
+        print(new_user)
+
+        quota = self.gi.quotas.create_quota(name="non_default_quota", description="testing",
+                amount="100 GB",
+                operation="+",
+                in_users=[new_user['username']],
```

Comment:
I think this needs to be either user ids or email addresses, see https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/services/quotas.py#L126

Reply:
Indeed